.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: php
+dist: trusty
```

Comment:
Consider using `trusty` only for legacy PHP versions:
```yml
matrix:
   include:
    - dist: trusty
      php: 5.4
    - dist: trusty
      php: 5.5
  allow_failuers:
    - php: master
```

Reply:
Also, the tests failed for `master`. Add `7.3` to the list of supported versions.

Reply:
@IonBazan Thank you for your suggestion!